definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-resp-journey-roles-and-permissions-nonprod.json

```diff
@@ -32,7 +32,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseFieldID": "RespondentOrganisationPolicy",
     "UserRole": "caseworker-divorce-superuser",
-    "CRUD": "R"
+    "CRUD": "CRU"
```

Comment:
So that we can edit the field in our "TEMP" event until we put the field in our workflow properly

Reply:
Good Marc. Can we put reverting this in a sub-story?
I'm very (maybe overly) scared of forgetting things. :)

Reply:
yep that's a good idea. If that's merged before https://tools.hmcts.net/jira/browse/DIV-6753 I could remove it as part of that ticket as DIV-6753 implements the respondentOrganisationField in the right way